### ISSUE TITLE
Changes to front end in Fava 1.25 break fava-envelope

### ISSUE THREAD

**Issue by BrendanKOz** [polarmutex/fava-envelope#75]
Fava-envelope 0.5.6 installed with Fava 1.24.4 working correctly.

Upgrade fava to 1.25, and Envelope Budget main screen fails to load.
Likely due to changes in front-end.

Error from Fava is generic.
Error from systemctl:

```
Jul 19 09:17:17 fava fava[4809]:     return _cached_url_for(endpoint, **values)
Jul 19 09:17:17 fava fava[4809]:   File "/usr/local/lib/python3.10/dist-packages/flask/helpers.py", line 256, in url_for
Jul 19 09:17:17 fava fava[4809]:     return current_app.url_for(
Jul 19 09:17:17 fava fava[4809]:   File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 2031, in url_for
Jul 19 09:17:17 fava fava[4809]:     return self.handle_url_build_error(error, endpoint, values)
Jul 19 09:17:17 fava fava[4809]:   File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 2020, in url_for
Jul 19 09:17:17 fava fava[4809]:     rv = url_adapter.build(  # type: ignore[union-attr]
Jul 19 09:17:17 fava fava[4809]:   File "/usr/local/lib/python3.10/dist-packages/werkzeug/routing/map.py", line 917, in build
Jul 19 09:17:17 fava fava[4809]:     raise BuildError(endpoint, values, method, self)
Jul 19 09:17:17 fava fava[4809]: werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'extension_report' with values ['bfile', 'currency', 'month', 'report_name']. Did you forget to specify values ['extension_name']?
```

**Comment by juandpinto** [comment]
Same issue here. Any update on this?

**Comment by JonBoyleCoding** [comment]
Still an issue as far as I'm aware.

**Comment by polarmutex** [comment]
should be fixed in 0.5.7

**Comment by BrendanKOz** [comment]
Verified working on:

- Beancount 2.3.5
- Fava 1.26
- Fava-envelope 0.5.8
